### PR TITLE
Expanded the supported LLM list

### PR DESCRIPTION
**Pull request by writinwaters**
### What problem does this PR solve?

_Briefly describe what this PR aims to solve. Include background context that will help reviewers understand the purpose of the PR._

### Type of change

- [x] Documentation Update

### CODE REVIEW

File: docs/guides/llm_api_key_setup.md [hunk 1]
```diff
@@ -10,10 +10,15 @@ You have two ways to input your LLM API key.
...
-> [ZHIPU-AI](https://open.bigmodel.cn/), [Moonshot](https://platform.moonshot.cn/docs)
+RagFlow supports the flowing LLMs, with more coming in the pipeline:
+
+- [OpenAI](https://platform.openai.com/login?launch)
+- [Tongyi-Qianwen](https://dashscope.console.aliyun.com/model), 
+- [ZHIPU-AI](https://open.bigmodel.cn/), 
+- [Moonshot](https://platform.moonshot.cn/docs)
+- [DeepSeek](https://platform.deepseek.com/api-docs/)
+- [Baichuan](https://www.baichuan-ai.com/home)
+- [Volcano Ark](https://www.volcengine.com/docs/82379)
```
Comment (KevinHuSh) [inline docs/guides/llm_api_key_setup.md]: VolcEngine

File: docs/guides/llm_api_key_setup.md [hunk 2]
```diff
@@ -10,10 +10,15 @@ You have two ways to input your LLM API key.
 ## Before Starting The System
 
 In **user_default_llm** of [service_conf.yaml](https://github.com/infiniflow/ragflow/blob/main/docker/service_conf.yaml), you need to specify LLM factory and your own _API_KEY_. 
-RagFlow supports the flowing LLM factory, and with more coming in the pipeline:
-
-> [OpenAI](https://platform.openai.com/login?launch), [Tongyi-Qianwen](https://dashscope.console.aliyun.com/model), 
-> [ZHIPU-AI](https://open.bigmodel.cn/), [Moonshot](https://platform.moonshot.cn/docs)
+RagFlow supports the flowing LLMs, with more coming in the pipeline:
```
Comment (JinHai-CN) [inline docs/guides/llm_api_key_setup.md]: RAGFlow

File: docs/guides/llm_api_key_setup.md [hunk 3]
```diff
@@ -10,10 +10,15 @@ You have two ways to input your LLM API key.
...
+
+- [OpenAI](https://platform.openai.com/login?launch)
```
Comment (JinHai-CN) [inline docs/guides/llm_api_key_setup.md:15]: Follow the same order above?

File: docs/quickstart.md [hunk 4]
```diff
@@ -101,8 +101,11 @@ RAGFlow is a RAG engine, and it needs to work with an LLM to offer grounded, hal
 
 - OpenAI
```
Comment (JinHai-CN) [inline docs/quickstart.md:102]: Follow the same order above?

**Review by JinHai-CN — APPROVED** [review]

**Pull request merged**